content/blog/deploying-convoy-to-fly.md

```diff
@@ -0,0 +1,210 @@
+
+[Fly.io](https://fly.io/) is a public cloud for running full stack applications and databases close to your users. It is designed to make it easy to deploy distributed and real-time apps, which makes it especially suitable for deploying your convoy instance and its dependencies. You can understand all Fly.io related concepts or commands that we use in this post by simply following along, but if you want to get a more in-depth background understanding of how things work on Fly, you can check the documentation [here](https://fly.io/docs/).
+
+## Deploy Dependencies
+
+You want to start out with deploying the dependencies first so that convoy can connect with them once it's deployed. To begin, you will need to install the Fly command line tool [flyctl](https://fly.io/docs/hands-on/install-flyctl/), create a Fly.io [account](https://fly.io/docs/hands-on/sign-up/), or [signin](https://fly.io/docs/hands-on/sign-in/) if you already own an account. Now that you're ready let's begin!
+
+### - Deploy and Configure Postgres: 
+
+Fly.io has an offering called "Fly Postgres", it's not the same as a managed Postgres database, it's rather another way to deploy your own self-managed Postgres DB on Fly with the benefits being that the creation process is automated, and you get some platform  integration to simplify management. We'll use this option in this guide.
+
+To create fly Postgres, run the following command on your fly.io authenticated machine:
+
+```
+fly postgres create
+```
+
+what follows is an interactive session where you are asked to:
+
+ 1. Select a name for your Postgres app. An example is `convoy-pg-data`, you will be asked to choose another name if an app on convoy already uses the name you have selected.
+
+ 2. Select an organization. The default organization **personal** is selected automatically if have not created any other organization.
+
+ 3. Select a region where you want Postgres to run.
+
+ 4. Select a plan. Choose a plan according to your current needs. Of course, the development plan should be preferred if you're only testing out things.
+
+ 5. You'd be asked if you want to Scale single node pg to zero after one hour. Again, choose according to your need.
+
+After these, the database will be deployed, copy and save your credentials as the output suggests. In a few moments, you will need to attach this Postgres deployment to your Convoy app.
+
+![output from creating a fly postgres instance](/blog-assets/static-ip-of-an-outbound-request.jpg)
```

Comment:
@Shulammite-Aso This image does not seem correct.

Reply:
fixed